File: src/main/java/no/nav/pto_schema/enums/arena/Hovedmaal.java
package no.nav.pto_schema.enums.arena;

public enum Hovedmaal {
    BEHOLDEA("BEHOLDEA"), // Beholde arbeid
    OKEDELT("OKEDELT"),   // Øke deltakelse
    SKAFFEA("SKAFFEA");   // Skaffe arbeid

    private final String kode;

    Hovedmaal(final String kode) {
        this.kode = kode;
    }

    @Override
    public String toString() {
        return kode;
    }
}

File: src/main/kotlin/no/nav/pto_schema/enums/arena/Hovedmaal.kt
package no.nav.pto_schema.enums.arena

enum class Hovedmaal(private val kode: String) {
    BEHOLDEA("BEHOLDEA"), // Beholde arbeid
    OKEDELT("OKEDELT"),   // Øke deltakelse
    SKAFFEA("SKAFFEA");   // Skaffe arbeid

    override fun toString(): String = kode
}

